aerospike/metadata.csv

```diff
@@ -469,6 +469,7 @@ aerospike.namespace.batch_sub_read_filtered_out,gauge,,transaction,,Number of ba
 aerospike.namespace.client_delete_filtered_out,gauge,,transaction,,Number of client delete transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,
 aerospike.namespace.client_udf_filtered_out,gauge,,transaction,,Number of client udf transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,
 aerospike.namespace.client_write_filtered_out,gauge,,transaction,,Number of client write transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,
+aerospike.namespace.client_read_filtered_out,gauge,,transaction,,Number of client read transactions that did not happen because the record was filtered out via a predicate expression.,0,aerospike,
```

Comment:
Any of the descriptions that start with the word `Number of`, I suggest changing to `The number of`.

Reply:
Updated for all metrics!